plugins/transport-grpc/src/main/java/org/opensearch/plugin/transport/grpc/proto/request/search/query/AbstractQueryBuilderProtoUtils.java

```diff
@@ -19,10 +19,32 @@
  */
 public class AbstractQueryBuilderProtoUtils {
 
+    private static QueryBuilderProtoConverterRegistry registry;
```

Comment:
Is it worth making an instance of this class that is instantiated with the registry and is passed around, to avoid having to manage static state and avoid needing to handle the case where the registry is unset?